modules/data.atmosphere/R/met.process.R

```diff
@@ -293,16 +295,21 @@ met.process <- function(site, input_met, start_date, end_date, model, host, dbpa
 
   #--------------------------------------------------------------------------------------------------#
   # Prepare for Model
+  # Determine output format name and mimetype
+  
+  model_info <- db.query(paste0("SELECT f.name, f.id, mt.type_string from modeltypes as m",
```

Comment:
There's nothing wrong with creating strings this way, but my personal preference is to use the `sprintf` function. Might be worth looking into if you have spare time.